include/pros/apix.h

```diff
@@ -550,7 +666,110 @@ int32_t fdctl(int file, const uint32_t action, void* const extra_arg);
  */
 #define DEVCTL_SET_BAUDRATE 17
 
+
+/******************************************************************************/
+/**                       RTOS STATIC TASK FACILITIES                        **/
+/******************************************************************************/
+
+#ifdef __cplusplus
+#define task_t pros::task_t
+#define task_fn_t pros::task_fn_t
+#define mutex_t pros::mutex_t
+#define sem_t pros::c::sem_t
+#define queue_t pros::c::queue_t
+#endif
+
+typedef uint32_t task_stack_t;
+
+/**
+ * Creates a task using statically allocated buffers. All tasks used by the PROS
+ * system must use statically allocated buffers.
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENOMEM - The stack cannot be used as the TCB was not created.
+ *
+ * \param function
+ *        Pointer to the task entry function
+ * \param parameters
+ *        Pointer to memory that will be used as a parameter for the task being
+ *        created. This memory should not typically come from stack, but rather
+ *        from dynamically (i.e., malloc'd) or statically allocated memory.
+ * \param prio
+ *        The priority at which the task should run.
+ *        TASK_PRIO_DEFAULT plus/minus 1 or 2 is typically used.
+ * \param stack_depth
+ *        The number of words (i.e. 4 * stack_depth) available on the task's
+ *        stack. TASK_STACK_DEPTH_DEFAULT is typically sufficienct.
+ * \param name
+ *        A descriptive name for the task.  This is mainly used to facilitate
+ *        debugging. The name may be up to 32 characters long.
+ *
+ * \return A handle by which the newly created task can be referenced. If an
+ * error occurred, NULL will be returned and errno can be checked for hints as
+ * to why task_create failed.
+ */
+task_t task_create_static(task_fn_t task_code, void* const param, uint32_t priority, const size_t stack_size,
+                          const char* const name, task_stack_t* const stack_buffer, static_task_s_t* const task_buffer);
+
+/**
+ * Creates a statically allocated mutex.
+ *
+ * All FreeRTOS primitives must be created statically if they are required for
+ * operation of the kernel.
+ *
+ * \param[out] mutex_buffer
```

Comment:
These buffers aren't out. They're the memory used run the type, so they're in. This applies to the buffers later as well.

Reply:
Fixed at cb96b0c